pages/common/deno.md

```diff
@@ -0,0 +1,25 @@
+# deno
+
+> A secure runtime for JavaScript and TypeScript
+> More information: <https://deno.land/>.
+
+- Run a JavaScript or TypeScript file:
+
+`deno file.ts`
+
+- Start a REPL (interactive shell):
+
+`deno`
+
+- Run a file with access to the network:
+
+`deno --allow-net server.ts`
+
+- Run a file from a URL:
+
+`deno https://deno.land/std/examples/welcome.ts`
+
+- Install an executable script from a URL:
+
+`deno install --allow-net --allow-read file_server https://deno.land/std/http/file_server.ts`
```

Comment:
I don't know what is `file_server` here exactly (a file?), but this should be changed just as suggested for the previous examples.

Reply:
`flie_server` here is the name of the exectuable after installing. After running the command, `file_server` is now installed on the user's machine and can be run from anywhere.

Reply:
Is that a name you can choose yourself? If yes, then it should be inside `{{ }}`.

Reply:
```suggestion
`deno install --allow-net --allow-read {{file_server}} {{https://deno.land/std/http/file_server.ts}}`
```
Just adding braces is fine here.